metabase/metabase
# Data Apps Confirm modal

## Description

Replaces the delete confirm modal, but this time, it can also be triggered by a custom action that has a `confirmMessage` set. (There is not currently any UI for users to set this, but I wanted to establish the codepath, so when we get a design for that, it should "just work" 🤞 )

Couple other minor changes:
- added some capitalization to modal titles
- added some special casing to the form submit buttons for `update` and `delete` implicit actions.
- deleted the old delete confirm modal (and some other code that depended on it, which is also deleted in #26062 )

![Screen Shot 2022-10-21 at 4 19 07 PM](https://user-images.githubusercontent.com/30528226/197297711-f1418bc6-f1ac-4157-ad16-6dab92fad840.png)


## Testing steps

- click a scaffolded delete button in a data app. you should get a delete confirm modal
- click delete and see that the record gets deleted 🔥
